packages/grafana-ui/src/components/MatchersUI/FieldsByFrameRefIdMatcher.test.tsx

```diff
@@ -109,4 +109,10 @@ describe('RefIDMultiPicker', () => {
     expect(mockOnChange).toHaveBeenLastCalledWith(['A', 'B']);
     /* eslint-enable testing-library/prefer-user-event */
   });
+
+  it('Should display a refID that does not exist in the selection', async () => {
+    multiSetup({ value: '/^(?:merge-A-B-C)$/' });
```

Comment:
These are nit-picks, but might make this nicer:

In the Github issue that you're fixing you say:
> Create two queries, anything is fine

I found this `merge-A-B-C` string to be slightly confusing without extra context to explain how we might end up here - what occurred to get us into this state.

Some ideas then:
1. Can we switch this string:
   1. to `merge-A-B`, as this reflects your github Issue more exactly (or update the Github issue)
   2. to simply
   ```suggestion
       multiSetup({ value: 'non-existent-refID' });
   ```
   This is clearer and simpler for the reader of this code. I'd go with this option personally.
2. Rearrange the `describe` blocks or setup code a little. The lines:
   ```ts
   const namesRegexp = /^(?:a|b \(ttt\)|bla\.foo|zzz\|cow|\$dollar\[baz\])$/;
   const namesArray = ['a', 'b (ttt)', 'bla.foo', 'zzz|cow', '$dollar[baz]'];
   ```
   are in the describe block with this test but not relevant to this test, which could be confusing later.